codespell_lib/data/dictionary.txt

```diff
@@ -5279,50 +5355,107 @@ assignenmet->assignment
 assignes->assigns
 assignmen->assignment, assign men,
 assignmenet->assignment
+assignmenets->assignments
 assignmens->assignments
 assignmet->assignment
+assignmetn->assignment
 assignmetns->assignments
+assignmets->assignments
 assignmnet->assignment
+assignmnets->assignments
 assignt->assign
+assignted->assigned
+assigntee->assignee
+assigntees->assignees
+assignting->assigning
 assigntment->assignment
+assigntments->assignments
+assignts->assigns
+assihn->assign
+assihned->assigned
+assihnee->assignee
+assihnees->assignees
+assihning->assigning
 assihnment->assignment
 assihnments->assignments
+assihns->assigns
 assime->assume
+assimed->assumed
+assimes->assumes
+assiming->assuming
+assimption->assumption
+assimptions->assumptions
 assimtote->asymptote
 assimtotes->asymptotes
 assined->assigned
+assinee->assignee
+assinees->assignees
 assing->assign
```

Comment:
By the way, `assing` could also be `gassing`, `massing`, `passing`, `sassing`. Maybe `assessing` too. Any such typos out there?

Reply:
I couldn't find any of these.